### PR TITLE
updated Dockerfile

### PR DESCRIPTION
**Pull request by tiagodevweb**
Congratulations for the repository, I made some small changes to reduce the size of the image, highlight the supervision in go-lang, I noticed other improvements but if I accepted this contribution I'll send it as much as possible.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,12 @@
-FROM alpine:3.14
+ARG ALPINE_VERSION=3.14
```
Comment (TrafeX) [inline Dockerfile:1]: Why did you make this a build arg?

File: Dockerfile [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+FROM alpine:${ALPINE_VERSION}
 LABEL Maintainer="Tim de Pater <code@trafex.nl>"
 LABEL Description="Lightweight container with Nginx 1.20 & PHP 8.0 based on Alpine Linux."
+# Setup document root
+WORKDIR /var/www/html
```
Comment (TrafeX) [inline Dockerfile:6]: Why did you move this to the top?

File: Dockerfile [hunk 3]
```diff
@@ -1,9 +1,12 @@
...
 
 # Install packages and remove default server definition
-RUN apk --no-cache add \
+RUN apk add --no-cache --update \
```
Comment (TrafeX) [inline Dockerfile:9]: Why are you adding the `--update` flag?

**Review by TrafeX — COMMENTED** [review]
Thank you for your contribution. I added some questions on why you're making a few changes.

**Comment by TrafeX** [comment]
Also, if you pull in the master branch the build should succeed.

File: Dockerfile (hunk 1 above)
Comment (tiagodevweb) [reply Dockerfile:1]: So that you can build an image with some specific version without changing the Dockerfile.

File: Dockerfile (hunk 2 above)
Comment (tiagodevweb) [reply Dockerfile:6]: By caching the layers and to avoid the mkdir of line 42.

File: Dockerfile (hunk 3 above)
Comment (tiagodevweb) [reply Dockerfile:9]: Update repository indexes.

File: Dockerfile (hunk 1 above)
Comment (TrafeX) [reply Dockerfile:1]: Good one, I like that :+1:

File: Dockerfile (hunk 2 above)
Comment (TrafeX) [reply Dockerfile:6]: Good improvement :+1:

File: Dockerfile (hunk 3 above)
Comment (TrafeX) [reply Dockerfile:9]: I think this is not needed since `--no-cache` already ignores the local cache if there was any.

Comment (tiagodevweb) [reply Dockerfile]: You're right, i already removed it

**Comment by TrafeX** [comment]
Thank you!
